pkg/utils/fileHandling.go

```diff
@@ -0,0 +1,82 @@
+package utils
```

Comment:
Don't put inside utils directory, put the file inside also use small case for file name without underscore and space